src/main/java/com/andagaev/parakeet/business/ConverterToDecimal.java

```diff
@@ -0,0 +1,22 @@
+package com.andagaev.parakeet.business;
+
+public class ConverterToDecimal {
+    public static int octalToDecimal(int number) {
```

Comment:
Move this function to `business/Convert` class

Reply:
Done